### PR TITLE
Validation: Added Missing Descriptions

### PR DESCRIPTION
**Pull request by dereklh7**
Added missing metadata descriptions.

### CODE REVIEW

File: BingMaps/rest-services/autosuggest.md [hunk 1]
```diff
@@ -558,14 +559,14 @@ This section briefly shows how to provide geographic parameters when sending a U
...
-![AutoSuggestCircleSmall](media/AutoSuggestCircleSmall.PNG)
+![Screenshot of a map of Seattle, Washington with a highlighted 100 meter radius of the 47.666673, -122.382242 coordinates.](media/AutoSuggestCircleSmall.PNG)
 
 #### Bounding Box Geographic Region
 
 Example: 
 > `userMapView=47.669424,-122.386053,47.664083,-122.378448`
 
-![AutoSuggestBoundingBoxSmall](media/AutoSuggestBoundingBoxSmall.PNG)
+![Screenshot of a map of Seattle, Washington with a highlighter perimeter of the 47.669424, -122.386053 coordinates.](media/AutoSuggestBoundingBoxSmall.PNG)
```
Comment (rflynn27) [inline BingMaps/rest-services/autosuggest.md:569]: *highlighted

**Comment by dereklh7** [comment]
@stevemunk @rbrundritt 

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

**Comment by stevemunk** [comment]
@dereklh7 Thank you. Please change value for 'ms.author' from 'v-chrfr' to 'eriklindeman', otherwise it should be ready to merge.

**Comment by dereklh7** [comment]
Hey @stevemunk - I've implemented the requested change, though I put "eriklind" as the value for the ms.author field, as that is the author's MS alias as needed for the ms.author field. Please let me know if there are any issues with this.

**Comment by stevemunk** [comment]
@dereklh7, no you are correct, my bad. Thank you.

**Pull request merged**